libs/domains/application/src/lib/slices/applications.slice.ts

```diff
@@ -5,13 +5,14 @@ import {
   createSlice,
   EntityState,
   PayloadAction,
+  Update,
 } from '@reduxjs/toolkit'
-import { ApplicationsApi, Application, ApplicationMainCallsApi } from 'qovery-typescript-axios'
+import { ApplicationsApi, Application, ApplicationMainCallsApi, Status } from 'qovery-typescript-axios'
 import { addOneToManyRelation, getEntitiesByIds, removeOneToManyRelation } from '@console/shared/utils'
 
 export const APPLICATIONS_FEATURE_KEY = 'applications'
 
-export interface ApplicationsState extends EntityState<Application> {
+export interface ApplicationsState extends EntityState<Application & { status?: Status }> {
```

Comment:
I think we should create `ApplicationEntity` so we can reuse this type in other places. Otherwise Typescript is not gonna be happy the day you want to use an Application with a status somewhere you have only Application as type.
We need to factories this thing because it's gonna be used throughout the project and it's gonna be updated quite often.